coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -162,14 +162,21 @@ def _extract_doc_comment(content, line, column, markers):
                     line, column and the extracted documentation. If not
                     matched, returns None.
     """
-    if markers[1] == '':
-        # Extract and align to start marker.
-        return _extract_doc_comment_simple(content, line, column, markers)
-    elif markers[1] == markers[2]:
-        # Search for the each-line marker until it runs out.
-        return _extract_doc_comment_continuous(content, line, column, markers)
-    else:
-        return _extract_doc_comment_standard(content, line, column, markers)
+    # This ignores fancy docstrings.
+    valid_docstring_regex = '\s*[' + markers[0] + '].*'
```

Comment:
ah okay you use `split()`. Though this looks super complicated, can't you just modify the begin regex here: https://github.com/coala/coala/pull/4383/commits/e44c88d5b00728c12eac417ca37170c4f0a3f50e#diff-df547e1526ff7317adef884f7414cdd8R247  ?